ioctl/cmd/account/account.go

```diff
@@ -131,6 +137,19 @@ func KsAccountToPrivateKey(signer, password string) (crypto.PrivateKey, error) {
 			return crypto.KeystoreToPrivateKey(account, password)
 		}
 	}
+
+	// find the account in pem files
+	pemFilePath := filepath.Join(config.ReadConfig.Wallet, "sm2sk-"+address.String()+".pem")
+	prvKey, err := crypto.ReadPrivateKeyFromPem(pemFilePath, password)
+	if err == nil {
+		return prvKey, nil
+	}
+
+	var pathError *os.PathError
+	if !errors.As(err, &pathError) {
```

Comment:
we don't need this error?

it failed to find in keystore, and no sm2 exist, so it's just "not match all keys in keystore", not failed to read sm2 key